### DISCUSSION TITLE
Consistent Machine Noise Being IDed as USVs

### DISCUSSION THREAD

**Discussion by JAbrams112** [DrCoffey/DeepSqueak#229] [Q&A]
Hi everyone, I could use some help if anyone has dealt with this situation: I am evaluating USVs of rats in operant boxes while they learn tasks. The fans and machine noise create consistent buzz at about 35kHz and about 60kHz, which unfortunately is being identified consistently by DeepSqueak as USVs. Aside from running these training sessions without any background fan/machine noise, has anyone dealt with this and have ideas for how to get cleaner audio for identification, or how to ask the program to ignore these sounds? 
![DSPic](https://github.com/user-attachments/assets/03bdf463-cb32-42fa-8522-70b46ab16faa)

**Comment by DrCoffey** [comment]
Unfortunately, noise like this looks a lot like a real USV and is well within the normal USV range. The best option here is to remove the source of noise.

**Reply by JAbrams112** [reply]
Thank you, Dr. Coffey. I thought that might be the case, and it makes sense. I love DeepSqueak, thank you for sharing this program!
